src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/jpa/LocalAdminAuthority.kt

```diff
@@ -0,0 +1,67 @@
+package uk.gov.justice.digital.hmpps.nomisuserrolesapi.jpa
+
+import org.hibernate.Hibernate
+import org.hibernate.annotations.Type
+import org.hibernate.annotations.Where
+import java.time.LocalDate
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.FetchType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.OneToMany
+import javax.persistence.Table
+
+@Entity
+@Table(name = "LOCAL_ADMIN_AUTHORITIES")
+data class LocalAdminAuthority(
+  @Id
+  @Column(name = "LOCAL_AUTHORITY_CODE", nullable = false)
+  val localAuthorityCode: String,
+
+  @Column(name = "DESCRIPTION", nullable = false)
+  val description: String,
+
+  @Column(name = "LOCAL_AUTHORITY_CATEGORY")
+  val category: String,
+
+  @Column(name = "ACTIVE_FLAG")
+  @Type(type = "yes_no")
+  val active: Boolean,
+
+  @Column(name = "EXPIRY_DATE")
+  val expiryDate: LocalDate? = null,
+
+  @OneToMany(fetch = FetchType.LAZY)
+  @JoinColumn(name = "LOCAL_AUTHORITY_CODE")
+  val allAdministeredUsers: List<LAAGeneralUser> = listOf(),
+
+  @OneToMany(fetch = FetchType.LAZY)
+  @JoinColumn(name = "LOCAL_AUTHORITY_CODE")
+  val allAdministrators: List<LAAAdminUser> = listOf(),
+
+  @OneToMany(fetch = FetchType.LAZY)
+  @JoinColumn(name = "LOCAL_AUTHORITY_CODE")
+  @Where(clause = "ACTIVE_FLAG = 'Y'")
+  val administeredUsers: List<LAAGeneralUser> = listOf(),
+
+  @OneToMany(fetch = FetchType.LAZY)
+  @JoinColumn(name = "LOCAL_AUTHORITY_CODE")
+  @Where(clause = "ACTIVE_FLAG = 'Y'")
+  val administrators: List<LAAAdminUser> = listOf(),
```

Comment:
I have made assumption that we don't get expired but active rows that should not be included